Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -903,6 +905,23 @@ Inspector::Protocol::ErrorStringOr<Ref<JSON::ArrayOf<Inspector::Protocol::CSS::C
     return cssProperties;
 }
 
+Inspector::Protocol::ErrorStringOr<Ref<JSON::ArrayOf<String>>> InspectorCSSAgent::getLoadedFonts()
+{
+    auto fontFamilyNames = JSON::ArrayOf<String>::create();
+    auto* domAgent = m_instrumentingAgents.persistentDOMAgent();
+
+    for (auto* document : domAgent->documents()) {
+        auto iterator = document->fonts().get().createIterator(document->scriptExecutionContext());
+
+        while (auto font = iterator.next()) {
+            if (font->status() == FontFace::LoadStatus::Loaded)
```

Comment:
is there a reason we have to wait for `FontFace::LoadStatus::Loaded`?  it seems like `family()` mainly looks at the CSS declaration to get a value

do we need to do anything for `FontFace::LoadStatus::Loading`?  i.e. what if the frontend calls this while some fonts are loading?  do we just drop those loading fonts entirely or would this get called again later?